app/src/main/java/com/flatstack/android/DI.java

```diff
@@ -44,14 +47,14 @@ public static OkHttpClient getHttpClient() {
 
     private static volatile Picasso picassoInstance = null;
 
-    public static Picasso getPicasso() {
+    public static @NonNull Picasso getPicasso(@NonNull Context context) {
         Picasso localInstance = picassoInstance;
```

Comment:
nothing for getCacheDir, but 

- [ ] Picasso should probably be passed an app Context